SparkPost/2web2ui
# AC-987: Feature comparison section

### What Changed
 - Added feature change section which lists features that would be removed or need to change before plan change

### How To Test
 - Go to new plan select
- Choose free or lower plan tier
 - Check that feature changes populate and should be acknowledged

### To Do
- [ ] Verify copy
